whois-api/src/test/java/net/ripe/db/whois/api/rest/TestTransactionDao.java

```diff
@@ -8,7 +8,7 @@
 import net.ripe.db.whois.common.dao.jdbc.JdbcRpslObjectDao;
 import net.ripe.db.whois.common.domain.BlockEvent;
 import net.ripe.db.whois.common.mail.EmailStatus;
-import net.ripe.db.whois.common.transaction.TransactionConfiguration;
+import net.ripe.db.whois.common.TransactionConfiguration;
```

Comment:
why moving out `transaction` package?

Reply:
because rest of configs are in base ackage